apps/web/src/services/landing/components/LandingGroupWorkspaces.vue

```diff
@@ -75,6 +76,11 @@ const handleClickShowAll = () => {
 const handleClickButtonGroupToggle = () => {
     state.isButtonGroupOpened = !state.isButtonGroupOpened;
 };
+const handleOpenOverlay = (workspaceGroupId:string) => {
```

Comment:
nitpick: `workspaceGroupId` << I don't think this is used!

Reply:
This is a parameter that must be used when connecting to the API in the future!